chore(accounts-management)_: update StatusNode initialization and start process

saledjenic opened the pull request:
Removed unused endpoints:
- DeleteImportedKey
- DeleteImportedKeyV2

Unnecessary instantiation of GetManager removed from the StatusNode, instead it's passed from GethStatusBackend. Unused accountsManager removed.

----
Reference to other accounts-management refactor PRs:
- https://github.com/status-im/status-go/pull/6605
- https://github.com/status-im/status-go/pull/6612
- https://github.com/status-im/status-go/pull/6640
- https://github.com/status-im/status-go/pull/6660
- https://github.com/status-im/status-go/pull/6668
- https://github.com/status-im/status-go/pull/6673
- https://github.com/status-im/status-go/pull/6686
- https://github.com/status-im/status-go/pull/6687
- https://github.com/status-im/status-go/pull/6692

saledjenic committed:
chore_: unused endpoints removed

Removed endpoints:
- DeleteImportedKey
- DeleteImportedKeyV2

status-im-auto commented:
### Jenkins Builds
<details>
<summary>Click to see older builds (87)</summary>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | e9c8830e | [#1](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6605/1/) | 2025-05-27 21:24:53 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250527-212134-e9c883-pr6605.aar) |
| :heavy_check_mark: | e9c8830e | [#1](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6605/1/) | 2025-05-27 21:24:59 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250527-212135-e9c883-pr6605.zip) |
| :heavy_check_mark: | e9c8830e | [#1](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6605/1/) | 2025-05-27 21:25:08 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250527-212137-e9c883-pr6605.zip) |
| :heavy_multiplication_x: | e9c8830e | [#1](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6605/1/) | 2025-05-27 21:26:27 | ~4 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6605/1//consoleText) |
| :heavy_check_mark: | e9c8830e | [#1](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6605/1/) | 2025-05-27 21:27:02 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250527-212140-e9c883-pr6605.zip) |
| :heavy_check_mark: | e9c8830e | [#1](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6605/1/) | 2025-05-27 21:27:44 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250527-212145-e9c883-pr6605.zip) |
| :heavy_check_mark: | e9c8830e | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6605/1/) | 2025-05-27 21:28:13 | ~6 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250527-212136-e9c883-pr6605.zip) |
| :heavy_multiplication_x: | e9c8830e | [#1](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6605/1/) | 2025-05-27 21:32:01 | ~10 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6605/1//consoleText) |
| :heavy_check_mark: | e9c8830e | [#2](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6605/2/) | 2025-05-28 06:50:46 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250528-064741-e9c883-pr6605.aar) |
| :heavy_check_mark: | e9c8830e | [#2](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6605/2/) | 2025-05-28 06:51:15 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250528-064746-e9c883-pr6605.zip) |
| :heavy_check_mark: | e9c8830e | [#2](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6605/2/) | 2025-05-28 06:51:56 | ~4 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250528-064753-e9c883-pr6605.zip) |
| :heavy_check_mark: | e9c8830e | [#2](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6605/2/) | 2025-05-28 06:52:17 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250528-064743-e9c883-pr6605.zip) |
| :heavy_check_mark: | e9c8830e | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6605/2/) | 2025-05-28 06:53:35 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250528-064744-e9c883-pr6605.zip) |
| :heavy_check_mark: | e9c8830e | [#2](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6605/2/) | 2025-05-28 06:54:26 | ~6 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250528-064747-e9c883-pr6605.zip) |
| :heavy_multiplication_x: | e9c8830e | [#2](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6605/2/) | 2025-05-28 06:59:55 | ~12 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6605/2//consoleText) |
| :heavy_multiplication_x: | e9c8830e | [#2](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6605/2/) | 2025-05-28 07:19:49 | ~31 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6605/2//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 0cc5fd8d | [#3](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6605/3/) | 2025-05-28 07:16:41 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250528-071349-0cc5fd-pr6605.zip) |
| :heavy_check_mark: | 0cc5fd8d | [#3](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6605/3/) | 2025-05-28 07:17:08 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250528-071347-0cc5fd-pr6605.aar) |
| :heavy_check_mark: | 0cc5fd8d | [#3](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6605/3/) | 2025-05-28 07:17:21 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250528-071353-0cc5fd-pr6605.zip) |
| :heavy_check_mark: | 0cc5fd8d | [#3](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6605/3/) | 2025-05-28 07:18:13 | ~4 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250528-071358-0cc5fd-pr6605.zip) |
| :heavy_check_mark: | 0cc5fd8d | [#3](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6605/3/) | 2025-05-28 07:19:15 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250528-071354-0cc5fd-pr6605.zip) |
| :heavy_check_mark: | 0cc5fd8d | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6605/3/) | 2025-05-28 07:20:34 | ~6 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250528-071350-0cc5fd-pr6605.zip) |
| :heavy_multiplication_x: | 0cc5fd8d | [#3](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6605/3/) | 2025-05-28 07:23:38 | ~9 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6605/3//consoleText) |
| :heavy_multiplication_x: | 0cc5fd8d | [#3](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6605/3/) | 2025-05-28 07:51:49 | ~31 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6605/3//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 4ee65279 | [#4](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6605/4/) | 2025-05-29 18:55:22 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250529-185326-4ee652-pr6605.aar) |
| :heavy_check_mark: | 4ee65279 | [#4](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6605/4/) | 2025-05-29 18:56:18 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250529-185327-4ee652-pr6605.zip) |
| :heavy_check_mark: | 4ee65279 | [#4](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6605/4/) | 2025-05-29 18:56:21 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250529-185332-4ee652-pr6605.zip) |
| :heavy_check_mark: | 4ee65279 | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6605/4/) | 2025-05-29 18:56:38 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250529-185328-4ee652-pr6605.zip) |
| :heavy_check_mark: | 4ee65279 | [#4](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6605/4/) | 2025-05-29 18:58:20 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250529-185333-4ee652-pr6605.zip) |
| :heavy_check_mark: | 4ee65279 | [#4](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6605/4/) | 2025-05-29 18:59:27 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250529-185336-4ee652-pr6605.zip) |
| :heavy_check_mark: | 4ee65279 | [#4](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6605/4/) | 2025-05-29 19:02:11 | ~8 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6605/4//consoleText) |
| :heavy_check_mark: | 4ee65279 | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6605/1/) | 2025-05-29 19:11:47 | ~18 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250529-185330-4ee652-pr6605.zip) |
| :heavy_multiplication_x: | 4ee65279 | [#4](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6605/4/) | 2025-05-29 19:19:07 | ~25 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6605/4//consoleText) |
| | | | | | | |
| :heavy_check_mark: | f667d265 | [#5](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6605/5/) | 2025-05-30 06:32:41 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250530-062952-f667d2-pr6605.zip) |
| :heavy_check_mark: | f667d265 | [#5](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6605/5/) | 2025-05-30 06:32:52 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250530-062956-f667d2-pr6605.zip) |
| :heavy_check_mark: | f667d265 | [#5](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6605/5/) | 2025-05-30 06:32:53 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250530-062951-f667d2-pr6605.aar) |
| :heavy_check_mark: | f667d265 | [#5](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6605/5/) | 2025-05-30 06:34:00 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250530-062956-f667d2-pr6605.zip) |
| :heavy_check_mark: | f667d265 | [#5](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6605/5/) | 2025-05-30 06:35:00 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250530-062953-f667d2-pr6605.zip) |
| :heavy_check_mark: | f667d265 | [#5](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6605/5/) | 2025-05-30 06:35:53 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250530-063002-f667d2-pr6605.zip) |
| :heavy_multiplication_x: | f667d265 | [#5](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6605/5/) | 2025-05-30 06:36:43 | ~6 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6605/5//consoleText) |
| :heavy_check_mark: | f667d265 | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6605/2/) | 2025-05-30 06:43:22 | ~13 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250530-062955-f667d2-pr6605.zip) |
| :heavy_multiplication_x: | f667d265 | [#5](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6605/5/) | 2025-05-30 06:54:46 | ~24 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6605/5//consoleText) |
| | | | | | | |
| :heavy_check_mark: | e478786e | [#6](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6605/6/) | 2025-05-30 07:39:24 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250530-073635-e47878-pr6605.zip) |
| :heavy_check_mark: | e478786e | [#6](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6605/6/) | 2025-05-30 07:39:29 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250530-073639-e47878-pr6605.zip) |
| :heavy_check_mark: | e478786e | [#6](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6605/6/) | 2025-05-30 07:39:34 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250530-073634-e47878-pr6605.aar) |
| :heavy_check_mark: | e478786e | [#6](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6605/6/) | 2025-05-30 07:40:35 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250530-073639-e47878-pr6605.zip) |
| :heavy_check_mark: | e478786e | [#6](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6605/6/) | 2025-05-30 07:41:52 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250530-073636-e47878-pr6605.zip) |
| :heavy_check_mark: | e478786e | [#6](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6605/6/) | 2025-05-30 07:42:27 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250530-073643-e47878-pr6605.zip) |
| :heavy_check_mark: | e478786e | [#6](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6605/6/) | 2025-05-30 07:43:30 | ~6 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6605/6//consoleText) |
| :heavy_check_mark: | e478786e | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6605/3/) | 2025-05-30 07:46:09 | ~9 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250530-073637-e47878-pr6605.zip) |
| :heavy_check_mark: | e478786e | [#6](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6605/6/) | 2025-05-30 08:11:26 | ~34 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6605/6//consoleText) |
| | | | | | | |
| :heavy_check_mark: | da5ab4e1 | [#7](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6605/7/) | 2025-05-30 14:58:23 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250530-145621-da5ab4-pr6605.aar) |
| :heavy_multiplication_x: | da5ab4e1 | [#7](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6605/7/) | 2025-05-30 14:59:08 | ~2 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6605/7//consoleText) |
| :heavy_check_mark: | da5ab4e1 | [#7](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6605/7/) | 2025-05-30 14:59:16 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250530-145623-da5ab4-pr6605.zip) |
| :heavy_check_mark: | da5ab4e1 | [#7](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6605/7/) | 2025-05-30 14:59:21 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250530-145626-da5ab4-pr6605.zip) |
| :heavy_check_mark: | da5ab4e1 | [#7](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6605/7/) | 2025-05-30 14:59:21 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250530-145625-da5ab4-pr6605.zip) |
| :heavy_check_mark: | da5ab4e1 | [#7](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6605/7/) | 2025-05-30 15:00:32 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250530-145630-da5ab4-pr6605.zip) |
| :heavy_check_mark: | da5ab4e1 | [#7](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6605/7/) | 2025-05-30 15:02:50 | ~6 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250530-145632-da5ab4-pr6605.zip) |
| :heavy_check_mark: | da5ab4e1 | [#7](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6605/7/) | 2025-05-30 15:04:20 | ~7 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6605/7//consoleText) |
| :heavy_check_mark: | da5ab4e1 | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6605/4/) | 2025-05-30 15:07:31 | ~11 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250530-145626-da5ab4-pr6605.zip) |
| | | | | | | |
| :heavy_check_mark: | 8f0ecd2f | [#8](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6605/8/) | 2025-05-30 16:22:42 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250530-162046-8f0ecd-pr6605.aar) |
| :heavy_check_mark: | 8f0ecd2f | [#8](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6605/8/) | 2025-05-30 16:23:19 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250530-162048-8f0ecd-pr6605.zip) |
| :heavy_check_mark: | 8f0ecd2f | [#8](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6605/8/) | 2025-05-30 16:23:49 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250530-162052-8f0ecd-pr6605.zip) |
| :heavy_check_mark: | 8f0ecd2f | [#8](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6605/8/) | 2025-05-30 16:23:57 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250530-162049-8f0ecd-pr6605.zip) |
| :heavy_check_mark: | 8f0ecd2f | [#8](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6605/8/) | 2025-05-30 16:24:59 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250530-162053-8f0ecd-pr6605.zip) |
| :heavy_multiplication_x: | 8f0ecd2f | [#8](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6605/8/) | 2025-05-30 16:25:18 | ~4 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6605/8//consoleText) |
| :heavy_check_mark: | 8f0ecd2f | [#8](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6605/8/) | 2025-05-30 16:28:14 | ~7 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250530-162058-8f0ecd-pr6605.zip) |
| :heavy_multiplication_x: | 8f0ecd2f | [#8](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6605/8/) | 2025-05-30 16:31:09 | ~10 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6605/8//consoleText) |
| :heavy_check_mark: | 8f0ecd2f | [#5](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6605/5/) | 2025-05-30 16:33:02 | ~12 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250530-162050-8f0ecd-pr6605.zip) |
| | | | | | | |
| :heavy_multiplication_x: | a03f5fe0 | [#9](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6605/9/) | 2025-06-02 05:35:17 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6605/9//consoleText) |
| :heavy_check_mark: | a03f5fe0 | [#9](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6605/9/) | 2025-06-02 05:35:48 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250602-053348-a03f5f-pr6605.aar) |
| :heavy_check_mark: | a03f5fe0 | [#9](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6605/9/) | 2025-06-02 05:36:44 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250602-053353-a03f5f-pr6605.zip) |
| :heavy_check_mark: | a03f5fe0 | [#9](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6605/9/) | 2025-06-02 05:37:02 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250602-053351-a03f5f-pr6605.zip) |
| :heavy_check_mark: | a03f5fe0 | [#9](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6605/9/) | 2025-06-02 05:38:10 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250602-053355-a03f5f-pr6605.zip) |
| :heavy_check_mark: | a03f5fe0 | [#9](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6605/9/) | 2025-06-02 05:38:47 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250602-053349-a03f5f-pr6605.zip) |
| :heavy_check_mark: | a03f5fe0 | [#9](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6605/9/) | 2025-06-02 05:39:39 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250602-053400-a03f5f-pr6605.zip) |
| :heavy_check_mark: | a03f5fe0 | [#9](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6605/9/) | 2025-06-02 05:41:59 | ~8 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6605/9//consoleText) |
| :heavy_check_mark: | a03f5fe0 | [#6](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6605/6/) | 2025-06-02 05:44:30 | ~10 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250602-053352-a03f5f-pr6605.zip) |
| | | | | | | |
| :heavy_check_mark: | e2daca9f | [#10](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6605/10/) | 2025-06-02 06:06:29 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250602-060431-e2daca-pr6605.aar) |
| :heavy_check_mark: | e2daca9f | [#10](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6605/10/) | 2025-06-02 06:07:25 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250602-060436-e2daca-pr6605.zip) |
| :heavy_multiplication_x: | e2daca9f | [#10](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6605/10/) | 2025-06-02 06:07:33 | ~2 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6605/10//consoleText) |
| :heavy_check_mark: | e2daca9f | [#10](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6605/10/) | 2025-06-02 06:07:34 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250602-060433-e2daca-pr6605.zip) |
| :heavy_check_mark: | e2daca9f | [#10](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6605/10/) | 2025-06-02 06:08:32 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250602-060438-e2daca-pr6605.zip) |
| :heavy_check_mark: | e2daca9f | [#10](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6605/10/) | 2025-06-02 06:09:25 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250602-060432-e2daca-pr6605.zip) |
| :heavy_check_mark: | e2daca9f | [#10](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6605/10/) | 2025-06-02 06:10:16 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250602-060441-e2daca-pr6605.zip) |
| :heavy_multiplication_x: | e2daca9f | [#10](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6605/10/) | 2025-06-02 06:12:08 | ~7 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6605/10//consoleText) |
| :heavy_check_mark: | e2daca9f | [#7](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6605/7/) | 2025-06-02 06:14:39 | ~10 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250602-060434-e2daca-pr6605.zip) |
</details>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | bcc677b6 | [#11](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6605/11/) | 2025-06-02 06:15:49 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250602-061351-bcc677-pr6605.aar) |
| :heavy_check_mark: | bcc677b6 | [#11](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6605/11/) | 2025-06-02 06:16:46 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250602-061356-bcc677-pr6605.zip) |
| :heavy_check_mark: | bcc677b6 | [#11](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6605/11/) | 2025-06-02 06:16:55 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250602-061355-bcc677-pr6605.zip) |
| :heavy_check_mark: | bcc677b6 | [#11](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6605/11/) | 2025-06-02 06:17:52 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250602-061359-bcc677-pr6605.zip) |
| :heavy_check_mark: | bcc677b6 | [#11](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6605/11/) | 2025-06-02 06:18:46 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250602-061352-bcc677-pr6605.zip) |
| :heavy_check_mark: | bcc677b6 | [#11](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6605/11/) | 2025-06-02 06:19:53 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250602-061401-bcc677-pr6605.zip) |
| :heavy_check_mark: | bcc677b6 | [#11](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6605/11/) | 2025-06-02 06:21:23 | ~7 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6605/11//consoleText) |
| :heavy_check_mark: | bcc677b6 | [#8](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6605/8/) | 2025-06-02 06:23:36 | ~8 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250602-061443-bcc677-pr6605.zip) |
| :heavy_check_mark: | bcc677b6 | [#11](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6605/11/) | 2025-06-02 06:39:48 | ~25 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6605/11//consoleText) |
| | | | | | | |
| :heavy_check_mark: | d97cadbd | [#12](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6605/12/) | 2025-06-02 13:26:57 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250602-132455-d97cad-pr6605.aar) |
| :heavy_check_mark: | d97cadbd | [#12](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6605/12/) | 2025-06-02 13:27:41 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250602-132456-d97cad-pr6605.zip) |
| :heavy_check_mark: | d97cadbd | [#12](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6605/12/) | 2025-06-02 13:27:51 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250602-132500-d97cad-pr6605.zip) |
| :heavy_check_mark: | d97cadbd | [#12](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6605/12/) | 2025-06-02 13:28:09 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250602-132458-d97cad-pr6605.zip) |
| :heavy_check_mark: | d97cadbd | [#12](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6605/12/) | 2025-06-02 13:29:06 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250602-132502-d97cad-pr6605.zip) |
| :heavy_check_mark: | d97cadbd | [#12](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6605/12/) | 2025-06-02 13:30:56 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250602-132507-d97cad-pr6605.zip) |
| :heavy_check_mark: | d97cadbd | [#12](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6605/12/) | 2025-06-02 13:32:43 | ~7 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6605/12//consoleText) |
| :heavy_check_mark: | d97cadbd | [#9](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6605/9/) | 2025-06-02 13:34:38 | ~9 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250602-132459-d97cad-pr6605.zip) |
| :heavy_check_mark: | d97cadbd | [#12](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6605/12/) | 2025-06-02 13:50:48 | ~25 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6605/12//consoleText) |

igor-sirotin commented on node/get_status_node.go:
```diff
@@ -334,7 +313,6 @@ func (n *StatusNode) stop() error {
 	n.appMetricsSrvc = nil
 	n.walletSrvc = nil
 	n.localNotificationsSrvc = nil
-	n.personalSrvc = nil
```
Looks like this makes the personal service unused, let's delete it then?
https://github.com/status-im/status-go/blob/f281ca0a805b5482fc071bf9760e26e18b02118d/services/personal/service.go#L13-L16

Though we keep using the `PublicAPI` of it, but we create it independently and use it in 2 places:
https://github.com/status-im/status-go/blob/e478786e7df17f74d80d966542d38d25e57565ef/api/geth_backend.go#L146 https://github.com/status-im/status-go/blob/4f4be1a78a0a8d3bae1ec627209ef3daa28663e9/services/ext/service.go#L400

Then I suppose we need to extract this type from the service? 🤔 
Not sure what's best here.

saledjenic replied:
> Looks like this makes the personal service unused, let's delete it then?

Yes, I was thinking about removing it.

saledjenic replied:
@igor-sirotin but just got a better idea, why don't we keep the service, and move api logic there (as it should be actually) and use the service in the code instead of api instance, and still provide correct endpoints. Going to do that change.

saledjenic replied:
@igor-sirotin the idea from my previous comment implemented, please have a look.

igor-sirotin commented on api/geth_backend.go:
```diff
@@ -2496,7 +2466,9 @@ func (b *GethStatusBackend) getVerifiedWalletAccount(address, password string) (
 	}
 
-	key, err := b.accountManager.VerifyAccountPassword(config.KeyStoreDir, address, password)
+	keystoreDirPath := filepath.Join(config.DataDir, config.KeyStoreDir)
```
Can you please give a quick explanation, why?

I know we have confusion about absolute/relative paths: https://github.com/status-im/status-go/issues/5563

E.g. we definitely make `KeyStoreDir` absolute here: https://github.com/status-im/status-go/blob/e478786e7df17f74d80d966542d38d25e57565ef/api/geth_backend.go#L2186
While the `b.rootDataDir` is coming from client here: https://github.com/status-im/status-go/blob/c5293eb8468583cfae318d1062e8673a370b2ee0/mobile/status.go#L116

saledjenic replied:
@igor-sirotin yes, the keystore is a relative path, but the thing was in `NewNodeConfig`, the issue was in `keyStoreDir = filepath.Join(dataDir, "keystore")` which is removed now, cause there we were setting an absolute path for the keystore dir and after that doing concatenation, and that caused a problem now, cause the real path for keystore dir was a real absolute path concatenated twice. For some reason, before these changes, because instantiation of 2 account managers that issue was masked.

igor-sirotin replied:
okay, I hope it's all correct now

igor-sirotin reviewed (commented):
2 questions, otherwise 👍

saledjenic committed:
chore(accounts-management)_: update StatusNode initialization and start process

Unnecessary instantiation of GetManager removed from the StatusNode, instead it's passed
from GethStatusBackend. Unused accountsManager removed.

igor-sirotin commented on go.mod:
```diff
@@ -266,11 +265,9 @@ require (
 	github.com/shirou/gopsutil v3.21.11+incompatible // indirect
 	github.com/shopspring/decimal v1.2.0 // indirect
 	github.com/spaolacci/murmur3 v1.1.0 // indirect
-	github.com/status-im/keycard-go v0.0.0-20200402102358-957c09536969 // indirect
```
lol, why was it here in the first place? 😄

igor-sirotin commented:
dunno where's this request coming from, no related files were changed. I will force-merge.

<img width="420" alt="image" src="https://github.com/user-attachments/assets/ab2ea5e9-c2e5-48e0-bb1e-a1c8543b04be" />
